lib/express-rate-limit.js

```diff
@@ -73,6 +73,7 @@ function RateLimit(options) {
 
             if (options.max && current > options.max) {
               options.onLimitReached(req, res, options);
+              res.setHeader('Retry-After', Math.ceil(options.windowMs / 1000));
```

Comment:
This should be conditioned on `options.headers`, and I was thinking it would be inside of the default `options.handler` so that it could be overridden.